tensorflow_hub/resolver.py

```diff
@@ -525,3 +530,19 @@ def _call_urlopen(self, request):
 
   def is_http_protocol(self, handle):
     return handle.startswith(("http://", "https://"))
+
+  def _maybe_disable_cert_validation(self):
```

Comment:
You're already setting `_context_` in `__init__` so I'd propose the change the method as follows:

```python
  def _maybe_disable_cert_validation(self):
    """Disables cert validation if TFHUB_DISABLE_CERT_VALIDATION is set.

    Checks whether certificate validation should be disabled when resolving an
    URL for downloading a model. This should only be done if the URL is
    trustworthy.
    """
    if os.getenv(
        _TFHUB_DISABLE_CERT_VALIDATION) == _TFHUB_DISABLE_CERT_VALIDATION_VALUE:
      self._context.check_hostname = False
      self._context.verify_mode = ssl.CERT_NONE
      logging.warning("Disabled certificate validation for resolving handles.")
```